help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -0,0 +1,222 @@
+
+An example is shown below:
+![Schedule segment export](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Browse Destinations](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Destinations Dataflow Run](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Destinations Activation Data](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Segment](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/segment.png)
+
+1. Login to the Salesforce Marketing Cloud website. Then navigate to the **[!DNL Audience Builder]** > **[!DNL Contact Builder]** > **[!DNL All contacts]** > **[!DNL Email]** page and check if the profiles from the segment have been added.
+![Salesforce Contacts](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/contacts.png)
+
+1. To check if any profiles have been updated, navigate to the **[!DNL Email]** page check if the attribute values for the profile from the segment have been updated. 
+![Salesforce Contacts](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/contact-detail.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+### Unknown errors encountered while pushing events to Salesforce Marketing Cloud {#unknown-errors}
+
+When checking a dataflow run, if you see the error message below, verify that the Mapping ID that you provided in [!DNL Salesforce CRM] for your Platform segment is valid and exists within [!DNL Salesforce CRM].
+![Error](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/error.png)
+
+## Additional resources {#additional-resources}
+
+* [Salesforce developer portal](https://developer.salesforce.com/)
+
+### Limits {#limits}
+
+* Dynamics 365 imposes certain [rate limits](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/rate-limiting.html).
```

Comment:
Looks like a minor error slipped in. Can you please update Dynamics 365 to Salesforce?